frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -102,8 +99,7 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
       setUpdatedProvisioner(updatedStorageClass?.provisioner);
     };
 
-    const handleVolumeMode: React.ReactEventHandler<HTMLInputElement> = (event) =>
-      setVolumeMode(event.currentTarget.value);
+    const handleVolumeMode = (value: string) => setVolumeMode(value);
     const handleAccessMode = (value: string) => setRestoreAccessMode(value);
```

Comment:
These helpers aren't necessary. Just use `setVolumeMode` and `setRestoreAccessMode` directly since they take the same parameter values.

Reply:
ack